libbeat/management/status/status.go

```diff
@@ -53,3 +53,11 @@ type WithStatusReporter interface {
 	// SetStatusReporter sets the status reporter
 	SetStatusReporter(reporter StatusReporter)
 }
+
+// NoopReporter is a bare reporter interface
+type NoopReporter struct{}
+
+// UpdateStatus does nothing
+func (NoopReporter) UpdateStatus(_ Status, _ string) {
+
+}
```

Comment:
```suggestion
// noopReporter is a bare reporter interface
type noopReporter struct{}

// UpdateStatus does nothing.
func (noopReporter) UpdateStatus(Status, string) {}
```

Reply:
Oh, nevermind, it looks like it does? We're calling it from packbeat, not `status`. Plus, I assumed other clients of the status interface might want to use it.

Reply:
Yeah, my thinking was that because it's a generic helper, it should probably go where the interface is defined, in case other libraries can make use of it.

Reply:
I've similarly thought in the past that it should not be up to consumers to add this, but that would be a separate PR and not for this package. When that's done the noop implementers of this interface can all be replaced with that one. Until then, this is a tiny addition which would become a breaking change to remove. So let's not add it here now.

> "A little copying is better than a little dependency."
>
> -- https://go-proverbs.github.io/